sdk/identity/identity/review/identity.api.md

```diff
@@ -64,10 +64,15 @@ export class ChainedTokenCredential implements TokenCredential {
 
 // @public
 export class ClientCertificateCredential implements TokenCredential {
-    constructor(tenantId: string, clientId: string, certificatePath: string, options?: TokenCredentialOptions);
+    constructor(tenantId: string, clientId: string, certificatePath: string, options?: ClientCertificateCredentialOptions);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
     }
 
+// @public
+export interface ClientCertificateCredentialOptions extends TokenCredentialOptions {
+    includeX5c?: boolean;
```

Comment:
My vote's for `sendCertificate` because I think it's clearer about the effect; "x5c" is an implementation detail.

Reply:
Is this how it is implemented in other languages?

Reply:
Let's merge this for now with this name. We're currently not consistent for this first preview, but we can update this in the next preview once we've had feedback and Arch Board review.